src/pages/docs/argo-cd/instances/automated-installation.md

```diff
@@ -0,0 +1,142 @@
+
+With the Octopus Argo CD Gateway being published as a helm chart, several options exist to install it through automated
+means:
+* scripted install using the helm cli
+* Terraform
+* Argo CD Application
+
+The Gateway requires the following configuration items which define how it connects to both OctopusDeploy an Argo CD:
+
+| Value                                  | Type | Required                  | Description                                                                                                                                                     |
+|----------------------------------------|---|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| registration.octopus.name              | string | Required                  | The unique name of this gateway, used within Octopus Deploy                                                                                                     |
+| registration.octopus.serverApiUrl      | string | Required                  | The URL of the Octopus Server that the gateway will register                                                                                                    |
+| registration.octopus.serverAccessToken | string | Required                  | The bearer or API token used for authentication during initial registration process with Octopus Server                                                         |
+| registration.octopus.spaceId           | string | Required                  | The unique id of the Octopus Deploy space in which the gateway should reside                                                                                    |
+| registration.argocd.webUiUrl           | string | Optional (default: null)  | The URL of your Argo CD instance's Web UI, used for generating deep-links within Octopus Deploy                                                                 |   
+| gateway.octopus.serverThumbprint       | string | Optional                  | Needed only if your Octopus server uses a self-signed certificate for its grpc endpoints. The thumbprint of the X509 certificate used by Octopus's GRPC service |
+| gateway.octopus.serverGrpcUrl          | string | Required                  | The URL at which Octopus Server expects to receive grpc traffic (grpc://your-server.com:8443)                                                                   |
+| gateway.octopus.plainText              | string | Optional (default: false) | True if Octopus' grpc endpoint is using un-encrypted connections, false otherwise.                                                                              |
+| gateway.argocd.serverGrpcUrl           | string | Required                  | The URL on which your Argo CD instance is listening for connections                                                                                             |
```

Comment:
Probably worth mentioning this is from within the cluster, and provide a sample value: grpc://argocd-server.argocd.svc.cluster.local